apps/spruce/src/pages/version/VersionTabs.tsx

```diff
@@ -235,18 +229,13 @@ export const VersionTabs: React.FC<VersionTabProps> = ({ version }) => {
     setSelectedTab: selectNewTab,
   });
   return (
-    <>
-      {tabIsActive[PatchTab.TestAnalysis] && (
-        <TestAnalysisTabGuideCue refEl={testAnalysisTabRef} />
-      )}
-      <StyledTabs
-        aria-label="Patch Tabs"
-        selected={selectedTab}
-        setSelected={selectNewTab}
-      >
-        {/* @ts-expect-error: FIXME. This comment was added by an automated script. */}
-        {activeTabs.map((t: string) => allTabs[t])}
-      </StyledTabs>
-    </>
+    <StyledTabs
```

Comment:
Not sure exactly why your changes are introducing this but I'm seeing an empty tab rendered that isn't on prod (version `67096bb4f3a665000769a76d` with prod data)
<img width="428" alt="image" src="https://github.com/user-attachments/assets/86b03439-07ec-4679-a754-4fcd49930860">